src/QMCWaveFunctions/TrialWaveFunction.h

```diff
@@ -326,6 +326,14 @@ class TrialWaveFunction
                            std::vector<ValueType>& ratios,
                            Matrix<ValueType>& dratio);
 
+  /** Used by SOECPComponent to do faster SOC evaluation with derivatives of ratio
+   */
```

Comment:
Could you document this more like evaluateDerivRatios explaining what is evaluated?

Reply:
updated